### PR TITLE
doc: add issue templates

### PR DESCRIPTION
**Pull request by andyzhangx**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
doc: add issue templates, copied from nfs csi driver.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/support.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+about: Ask questions about this project
+
+---
+
+<!-- 
+STOP -- PLEASE READ!
+
+GitHub is not the right place for support requests.
+
+If you're looking for help, post your question on the [Kubernetes Slack ](http://slack.k8s.io/) Sig-Storage Channel.
```
Comment (pohly) [inline .github/ISSUE_TEMPLATE/support.md:12]:
```suggestion
If you're looking for help, post your question on the [Kubernetes Slack ](http://slack.k8s.io/) #sig-storage Channel.
```

Comment (pohly) [reply .github/ISSUE_TEMPLATE/support.md:12]: I think the official spelling is "SIG Storage" (capital SIG, no hyphen). At least that was most common among SIGs the last time I checked.

**Comment by xing-yang** [comment]
Looks good other than Patrick's comment.

**Comment by andyzhangx** [comment]
> Looks good other than Patrick's comment.

done, PTAL

**Review by pohly — APPROVED** [review]
/lgtm

**Pull request merged**